src/lib/features/feature-toggle/tests/feature-toggles.e2e.test.ts

```diff
@@ -3707,3 +3707,13 @@ test('can get evaluation metrics', async () => {
         ],
     });
 });
+
+test('Should not be able to create flag with random type', async () => {
```

Comment:
Can we be clearer what we mean here? Unleash doesn't have a flag type called random, but it _might_.

```suggestion
test("Should not be able to create flag with a type that doesn't exist", async () => {
```